prow/jobs/bundles/bundles.yaml

```diff
@@ -0,0 +1,44 @@
+job_template: &job_template
+  skip_report: true
+  decorate: true
+  path_alias: github.com/kyma-project/bundles
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/bundles"
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-bundles-git-token: "true"
+
+presubmits: # runs on PRs
+  kyma-project/bundles:
+    - name: bundles
+      run_if_changed: "^bundles/"
+      branches:
+        - master
+      <<: *job_template
+      labels:
+        <<: *job_labels_template
+        preset-build-pr: "true"
+
+postsubmits:
```

Comment:
how this configuration refers to https://github.com/kyma-project/bundles/blob/master/docs/releasing.md
? 
how do we handle tagging repository?